lib/src/dartdevc/module_reader.dart

```diff
@@ -131,4 +131,19 @@ class ModuleReader {
       return modules;
     });
   }
+
+  /// Invalidates all [Module]s for [package].
+  void invalidatePackage(String package) {
+    // `map` must be of type <AssetId|ModuleId, dynamic>
+    removePackage(String package, Map map) {
```

Comment:
I'd either avoid passing `package` here, since it's already in the closure scope, or make `_removePackage` a private method rather than a local function.